vagrant/vagrant-test-sanitizers.sh

```diff
@@ -54,6 +59,10 @@ if [[ $NSPAWN_EC -eq 0 ]]; then
     rsync -amq /var/tmp/systemd-test*/journal "$LOGDIR/TEST-01-BASIC_sanitizers-qemu/" &>/dev/null || :
 fi
 
+exectask "systemd-networkd_sanitizers" \
+            "test/test-network/systemd-networkd-tests.py --build-dir=$PWD/build --debug --asan-options=$ASAN_OPTIONS --ubsan-options=$UBSAN_OPTIONS" \
+            1 # Ignore this task's exit code
```

Comment:
I should probably have been more specific about what I meant by "look for ASan/UBsan reports". After the test is finished, the script should automatically make sure ASan or UBsan hasn't reported anything (because we can't expect anyone to look at the logs manually every time PRs are opened and generally nobody pays attention to ever-green statuses). The journal part can be partly borrowed from https://github.com/systemd/systemd/blob/33fe9e3fd054e70a1f9e5af359bfe3dd8435ce67/test/test-functions#L578-L592. The same regular expression can probably be used to look for ASan/UBsan reports in the output of the test.

Reply:
Ugh, that definitely makes sense, I'm not sure why I didn't think of this earlier. Thanks! :-)